spec/features/admin/adjustments_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 require "spec_helper"
 
-feature '
+describe '
```

Comment:
I think we can't have specs using `spec_helper` with `describe`, as they will be run as system specs, but will break because they are not calling the necessary helper (`system_helper`).

Reply:
I'm noticing this as the build is failing here, on `background`:

https://github.com/openfoodfoundation/openfoodnetwork/pull/8179/files#diff-0f3532dd56c5ea48814fa945b58cd9a5851a2ddc631cd91020d85e25d783f1e8R319

Which passes on `feature specs` but `breaks system` specs.